executor/adapter.go

```diff
@@ -248,8 +248,9 @@ func (a *ExecStmt) Exec(ctx context.Context) (sqlexec.RecordSet, error) {
 	}
 
 	var txnStartTS uint64
-	if sctx.Txn(false).Valid() {
-		txnStartTS = sctx.Txn(true).StartTS()
+	txn, _ := sctx.Txn(false)
```

Comment:
It is better to check the return error, I know the error will be not nil because `inactive = false`, but we maybe will change the Txn implement in the future.